zyzmoz/global-player
# feature: Create LineChart Component

#### **Trello ticket URL:**
[Ticket](https://trello.com/c/CbN86Bn5/80-create-linechart-component)


#### **Description:**
Create a Line Chart to show data about two players


**Run command:**

```
npm install \
npm run start
```

- [x] This code includes new npm packages/libraries

#### **QA instructions (if applicable):** 
1. Go to Storybook
2. Check is the component is rendered correctly

---

_Thank you for your contribution to the Global Player monorepo._
_Before submitting this PR, please make sure:_

- [ ] Unit tests (if needed)
- [x] Code cleanup (removed dead code, console logs, sorted props, etc)
- [x] Updated Trello ticket with info and linked to this PR
